### DISCUSSION TITLE
Key Prefix in serverside getFixedT is missing

### DISCUSSION THREAD

**Discussion by jordiyapz** [sergiodxa/remix-i18next#141] [Q&A]
Hi, sorry I'm new to this,

I cannot find an option to add prefix for key in `TFunction` returned by `i18nextt.getFixedT` function where i18nextt is an instance of `RemixI18Next`. From the original `i18next` we have `keyPrefix` option at the 3rd argument of `getFixedT`. And also the client-side hook `useTranslation` also have it. 

Is it really missing in the server-side?

**Comment by sergiodxa** [comment]
Yes, I didn't implemented it, I'm going to create an issue to do it.
